components/icu4x_ecma402/src/pluralrules.rs

```diff
@@ -0,0 +1,163 @@
+//! TODO: License blurb.
+
+//! Implements the traits found in [ecma402_traits::pluralrules].
+
+use ecma402_traits;
+use icu_pluralrules as ipr;
+use icu_pluralrules::PluralRulesError;
+
+pub(crate) mod internal {
+    use ecma402_traits::pluralrules::options::Type;
+    use ecma402_traits::pluralrules::Options;
+    use icu_pluralrules::{PluralCategory, PluralOperands, PluralRuleType};
+
+    /// Converts the trait style option to an equivalent ICU4X type.
+    pub fn to_icu4x_type(style: &Type) -> PluralRuleType {
+        match style {
+            Type::Ordinal => PluralRuleType::Ordinal,
+            Type::Cardinal => PluralRuleType::Cardinal,
+        }
+    }
+
+    /// Converts the number to format into the operands representation.
+    pub fn to_icu4x_operands(n: f64, _opts: Options) -> PluralOperands {
+        // TODO: This does not convert the display Options correctly but
+        // probably should.
+        let nstr = format!("{}", n);
+        nstr.parse().unwrap()
+    }
+
+    /// Expresses the [PluralCategory] as a `str`.
+    pub fn as_str(c: PluralCategory) -> &'static str {
+        match c {
+            PluralCategory::Few => "few",
+            PluralCategory::Many => "many",
+            PluralCategory::One => "one",
+            PluralCategory::Other => "other",
+            PluralCategory::Two => "two",
+            PluralCategory::Zero => "zero",
+        }
```

Comment:
You could use `Debug` trait here to get the name

Reply:
I am unsure as to whether debug fixtures in general should be used for purposes other than actual debugging.